### ISSUE TITLE
False positives for Squiz.PHP.CommentedOutCode.Found

### ISSUE THREAD

**Issue by paulschreiber** [WordPress-Coding-Standards/WordPress-Coding-Standards#501]
This line is being flagged as commented-out code
```
	} // foreach $region
```

It's pretty common to do line comments at the end of `if` and `for` blocks like that. Please don't flag them.

Here's the whole block:
```php
	<?php
	foreach ( $quadrant as $region => $restaurants ) {
		// Grab the region color, used to get the winner id
		$region_color = $region_colors[ $region ];
		// Grab the winner id, used for the r_round_url
		$round_winner_id = self::round_winner_id( $quadrant_name, $region_color, $region );
	?>
	<div class="region-reviews <?php echo esc_attr( self::round_status( $quadrant_name, null, $region ) ), esc_attr( $region ); ?>">
....
	</div>
	<!-- .post -->
	<?php
	} // foreach $region
	echo '</div>';
	$i++;
} // foreach $quadrant
```

**Comment by paulschreiber** [comment]
This belongs in PHPCS. Closing.

**Comment by westonruter** [comment]
@paulschreiber you can customize the percentage threshold for when a comment is considered commented-out code or not. See [`Squiz_Sniffs_PHP_CommentedOutCodeSniff::$maxPercentage`]( https://github.com/squizlabs/PHP_CodeSniffer/blob/ed257ca0e56ad86cd2a4d6fa38ce0b95141c824f/CodeSniffer/Standards/Squiz/Sniffs/PHP/CommentedOutCodeSniff.php#L43-L48)

**Comment by paulschreiber** [comment]
The problem isn't the threshold — it's that PHPCS is thinking too many things — things that are not code — are code. Example:
```
// default
```

**Comment by GaryJones** [comment]
If the body of a `foreach` is so long that one needs a comment to remind themselves what a closing `}` is for, then it's a code smell that the body is doing too much and should be abstracted into a function (maybe switch to `array_walk()`).

IMO, there's no reason, ever, for a comment on a closing `}`, that simply indicates what PHP structure it is closing.

**Comment by paulschreiber** [comment]
Even with short methods, adding comments to the end of `if`, `for`, `try` and other blocks helps improve the readability of the code.

**Comment by GaryJones** [comment]
> Even with short methods, adding comments to the end of if, for, try and other blocks helps improve the readability of the code.

I think we're going to fundamentally disagree there, as is our right :-) For me, short methods that are not indented more than one or two levels, are obvious for what closing braces are for (the indentation matches the `if`, `for`, et al anyway. Having `// end foo` just adds visual clutter.
